grpc/request.go

```diff
@@ -0,0 +1,54 @@
+package grpc
+
+import (
+	"errors"
+
+	"github.com/gojek/fiber"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/protobuf/proto"
+)
+
+type Request struct {
+	Metadata       metadata.MD
+	RequestPayload proto.Message
+	ResponseProto  proto.Message
+	ServiceMethod  string
```

Comment:
`ServiceMethod` and `hostport` do not feel natural as part of `Request` properties. I feel that those 2 shall be part of dispatcher property.